### ISSUE TITLE
rm --stop is not working in compose 1.11.2 (Docker for Mac 17.03.1-ce-mac5)

### ISSUE THREAD

**Issue by djbingham** [docker/compose#4774]
Hi.

I'm not sure whether this is a bug in the software, or a mistake in the online documentation. Looking at the online docs, I saw that `docker-compose rm` should have an option `-s` (or `--stop`) to auto-stop the containers, if necessary, before attempting to remove them. This seemed like a handy shortcut, so I gave it a go and received the following output, which seems to suggest that option is not available:

> Removes stopped service containers.
> 
> By default, anonymous volumes attached to containers will not be removed. You
> can override this with `-v`. To list all volumes, use `docker volume ls`.
> 
> Any data which is not in a volume will be lost.
> 
> Usage: rm [options] [SERVICE...]
> 
> Options:
>     -f, --force   Don't ask to confirm removal
>     -v            Remove any anonymous volumes attached to containers
>     -a, --all     Deprecated - no effect.

I'm using the latest version of Docker for Mac (17.03.1-ce-mac5, which has Compose 1.11.2).

**Comment by shin-** [comment]
That option was introduced in 1.12.0. Please upgrade =)

**Comment by djbingham** [comment]
Hi, thanks for the quick reply. Is 1.12 available just in Docker for Mac
beta at the moment? Docker for Mac insists that it's already fully up to
date on my machine.

**Comment by zot24** [comment]
Same issue here, I'm using Docker for Mac that install `docker 17.03.1-ce` however I'm using `docker-compose 1.13.1` and getting the following error:

```
No such service: <Container: data_app_1 (781983)>
```

when running:

```
docker-compose rm -v --force --stop
```

Not sure if the fact that I'm using a containerised version of `docker-compose` could affect this behaviour, the full command looks like:

```
docker run --rm \
    -v ~/.docker/config.json:/root/.docker/config.json \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v $(pwd):/data \
    -w /data \
    registry.zot24.com/zot24/docker-compose:1.13.0 \
    \
    docker-compose rm -v --force --stop
```

> Note: On my Mac I got `docker-compose 1.11.2` which is the version that Docker for Mac install by default

**Comment by shin-** [comment]
@zot24 It will be fixed by #4860 in the next (1.14.0) release. Sorry for the inconvenience!
